modules/migration-understanding-data-copy-methods.adoc

```diff
@@ -3,31 +3,33 @@
 // migration/migrating-3-4/migrating-openshift-3-to-4.adoc
 // migration/migrating-4-4/migrating-openshift-4-to-4.adoc
 [id='migration-understanding-data-copy-methods_{context}']
-= Understanding the file system and snapshot data copy methods
+= Understanding the data copy methods for migration
 
-The CAM tool supports the file system and snapshot data copy methods for migrating data from the source cluster to the target cluster.
+The CAM tool supports the file system and snapshot data copy methods for migrating data from the source cluster to the target cluster. You can select a method that is suited for your environment and is supported by your storage provider.
```

Comment:
I know we use `file system` and `filesystem` interchangeably throughout a lot of our docs. You have it as one word when matching the UI previously, so I suggest making one word here, too. But this would require changing it through the rest of the page.

Reply:
`file system` is correct, according to the IBM Style Guide, but the UI has `filesystem`. My compromise was to use `file system` unless specifically referring to the UI.